src/functions/commerce/provider.php

```diff
@@ -48,4 +48,36 @@ function tribe_tickets_commerce_is_test_mode() {
 	 * @todo This method likely should be focused on paypal only usage as it's gateway specific conditional.
 	 */
 	return \TEC\Tickets\Commerce\Gateways\PayPal\Gateway::is_test_mode();
+}
+
+/**
+ * Determine whether the legacy TribeCommerce should be shown or not.
+ *
+ * @since TBD
+ *
+ * @return boolean
+ */
+function tec_tribe_commerce_is_available() {
+
+	if ( defined( 'TEC_TRIBE_COMMERCE_ENABLED' ) ) {
+		return (bool) TEC_TRIBE_COMMERCE_ENABLED;
+	}
+
+	$env_var = getenv( 'TEC_TRIBE_COMMERCE_ENABLED' );
+
+	if ( false !== $env_var ) {
+		return (bool) $env_var;
+	}
+
+	// Todo, @juanfra @rafsuntaskin decide the proper version.
+	$active = tribe_installed_before( 'Tribe__Tickets__Main', '5.1' );
```

Comment:
Lets not use versions, lets make it based on a cached variable of someone having the actual active account from PayPal IPN.